modules/dynamic-provisioning-storage-class-definition.adoc

```diff
@@ -23,7 +23,7 @@ metadata:
   annotations: <4>
     storageclass.kubernetes.io/is-default-class: 'true'
     ...
-provisioner: kubernetes.io/aws-ebs <5>
+provisioner: kubernetes.io/amazon-ebs <5>
```

Comment:
I don't think the provisioner should be updated. Needs confirm from OCP storage QE.

Reply:
@duanwei33 Could you please confirm? Thank you!

Reply:
Yes, kubernetes.io/aws-ebs should not be changed to kubernetes.io/amazon-ebs. The storage doesn't work with kubernetes.io/amazon-ebs.

Reply:
@duanwei33 Could you explain a bit further why we can't use the correct AWS service name here?

Reply:
@codymant Hi, `kubernetes.io/aws-ebs` is provisioner name in kubernetes for dynamic provisioning, `kubernetes.io/amazon-ebs` will not be detected as below:

>   Warning  ProvisioningFailed   <invalid> (x2 over <invalid>)  persistentvolume-controller  no volume plugin matched name: kubernetes.io/amazon-ebs

Reply:
As discussed, revoking the change.